barretenberg/cpp/src/barretenberg/plonk/proof_system/widgets/transition_widgets/elliptic_widget.hpp

```diff
@@ -78,8 +78,9 @@ template <class Field, class Getters, typename PolyContainer> class EllipticKern
     inline static std::set<PolynomialIndex> const& get_required_polynomial_ids()
     {
         static const std::set<PolynomialIndex> required_polynomial_ids = {
-            PolynomialIndex::Q_1, PolynomialIndex::Q_3, PolynomialIndex::Q_4, PolynomialIndex::Q_ELLIPTIC,
-            PolynomialIndex::W_1, PolynomialIndex::W_2, PolynomialIndex::W_3, PolynomialIndex::W_4
+            PolynomialIndex::Q_1, PolynomialIndex::Q_3,        PolynomialIndex::Q_4,
+            PolynomialIndex::Q_M, PolynomialIndex::Q_ELLIPTIC, PolynomialIndex::W_1,
```

Comment:
Q_M has been added because the elliptic doubling gate is now encoded using two selector values, instead of one q_elliptic selector value.